### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by xtreme-sameer-vohra**
## Proposed Changes 
Fix metrics collector yaml path

### CODE REVIEW

File: docs/admin/collecting-metrics/README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ In the following example, you can configure a single collector instance using a
 1. Create a Deployment, Service, and ConfigMap for the collector by entering the following command:
 
        ```bash
-       kubectl apply -f https://raw.githubusercontent.com/knative/docs/main/docs/admin/collecting-metrics/collector.yaml
+       kubectl apply -f https://raw.githubusercontent.com/knative/docs/master/docs/admin/collecting-metrics/collector.yaml
```
Comment (abrennan89) [inline docs/admin/collecting-metrics/README.md:46]: this isn't right afaik, we use the main branch now?

Comment (abrennan89) [reply docs/admin/collecting-metrics/README.md:46]: cc @skonto if there's any metrics stuff that's still valid and isn't on the `main` branch it should move

Comment (skonto) [reply docs/admin/collecting-metrics/README.md:46]: The file also exists here: https://raw.githubusercontent.com/knative/docs/main/docs/admin/collecting-metrics/collector.yaml. We use main now. I am not aware of anything to be moved.

**Comment by skonto** [comment]
@carlisia hi, I have tested the config on main branch https://github.com/knative/serving/issues/11822#issuecomment-914262577 and it worked. @xtreme-sameer-vohra  why you want to use master (afaik all Knative repos renamed master to main)?

**Comment by carlisia** [comment]
I just realized that's my boss, hi!! 🌊 😆

**Comment by xtreme-sameer-vohra** [comment]
Whoops, sorry for the confusion folks. 
@abrennan89 is absolutely right about `main` being the standard branch and as @skonto shared the current link works so TBH I am not entirely sure how I ran into the problem in the first place 😥 or if it was all just in my head 😵‍💫

**Pull request closed without merging**